versions.tf

```diff
@@ -20,12 +20,12 @@ terraform {
 
     google = {
       source  = "hashicorp/google"
-      version = "~> 3.53"
+      version = ">= 3.53, < 5.0"
     }
   }
 
   provider_meta "google" {
-    module_name = "blueprints/terraform/terraform-google-lb-internal/v4.2.0"
+    module_name = "blueprints/terraform/terraform-google-lb-internal/v4.3.0"
```

Comment:
This will be auto updated by our release PR

Reply:
ok, reverting